scripts/recover_backups.py

```
@@ -0,0 +1,253 @@
+                summary = data['input'].get('summary', '')[:60]
+                who = data['input'].get('who', 'unknown')
+                print(f"  - {f.name}")
+                print(f"    {ts} | {who} → {summary}")
+            except Exception as e:
+                print(f"  - {f.name} (ERROR: {e})")
+        return
+    
+    # Check database exists
+    if not GRAPH_DB.exists():
+        print(f"❌ Database not found: {GRAPH_DB}")
+        print(f"Run: python3 -c 'from openclaw_hooks.nima_memory.index import initDatabase; initDatabase()'")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)